perl/bin/bb_vcf_to_ascat_cn.pl

```diff
@@ -0,0 +1,78 @@
+#!/software/perl-5.16.2/bin/perl
+
+#HACK do not commit!!
+##!/usr/bin/perl
+
+##########LICENCE##########
+# Copyright (c) 2016 Genome Research Ltd.
```

Comment:
Done, also updated for battenberg.pl. There are probably other files that also need this updating.